### ISSUE TITLE
nix expressions for macOS broken

### ISSUE THREAD

**Issue by plumps** [flyingcircusio/vulnix#11]
```
➜  vulnix git:(master)  nix-build .
```

```
downloading ‘https://github.com/NixOS/nixpkgs-channels/archive/453086a15fc0db0c2bc17d98350b0632551cb0fe.tar.gz’... [0/0 KiB, 0.0 KiB/s]
these derivations will be built:
  /nix/store/2x5rfglpbj27fa63fg1abv3jij4zbsbj-python3.5-execnet-1.4.1.drv
  /nix/store/xi48khw9p49xnjxb569d8srdds1wy0p2-python3.5-mccabe-0.4.0.drv
  /nix/store/hc4nazq871ksdzrvn23mdsz76smpg34z-python3.5-flake8-2.5.4.drv
  /nix/store/pkjrkrl874z6dfwj57j545igkc94k04h-python3.5-pytest-cache-1.0.drv
  /nix/store/v48hcg7s73qamrlsksmf6j3lv8qarxbh-python3.5-vulnix-0.1.dev0.drv
building path(s) ‘/nix/store/5yh2y2r8nzzmh41122njlnvngrpmi1zr-python3.5-execnet-1.4.1’
unpacking sources
unpacking source archive /nix/store/g739xbqq74v0hx5dswn4nrj40q9vy4g0-execnet-1.4.1.tar.gz
source root is execnet-1.4.1
setting SOURCE_DATE_EPOCH to timestamp 1441220062 of file execnet-1.4.1/setup.cfg
patching sources
configuring
building
Download error on https://pypi.python.org/simple/setuptools_scm/: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:645) -- Some packages may not be found!
Download error on https://pypi.python.org/simple/setuptools-scm/: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:645) -- Some packages may not be found!
Couldn't find index page for 'setuptools_scm' (maybe misspelled?)
Download error on https://pypi.python.org/simple/: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:645) -- Some packages may not be found!
No local packages or download links found for setuptools-scm
Traceback (most recent call last):
  File "nix_run_setup.py", line 6, in <module>
    exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\\r\\n', '\\n'), __file__, 'exec'))
  File "setup.py", line 68, in <module>
    main()
  File "setup.py", line 64, in main
    setup_requires=['setuptools_scm'],
  File "/nix/store/p5q8daa5msqgkj7di16vch322ial2sh8-python3-3.5.1/lib/python3.5/distutils/core.py", line 108, in setup
    _setup_distribution = dist = klass(attrs)
  File "/nix/store/86hcp0sb3ml9wfdfgapzn5bdvy1darh9-python3.5m-setuptools-19.4/lib/python3.5/site-packages/setuptools-19.4-py3.5.egg/setuptools/dist.py", line 269, in __init__
  File "/nix/store/86hcp0sb3ml9wfdfgapzn5bdvy1darh9-python3.5m-setuptools-19.4/lib/python3.5/site-packages/setuptools-19.4-py3.5.egg/setuptools/dist.py", line 313, in fetch_build_eggs
  File "/nix/store/86hcp0sb3ml9wfdfgapzn5bdvy1darh9-python3.5m-setuptools-19.4/lib/python3.5/site-packages/setuptools-19.4-py3.5.egg/pkg_resources/__init__.py", line 827, in resolve
  File "/nix/store/86hcp0sb3ml9wfdfgapzn5bdvy1darh9-python3.5m-setuptools-19.4/lib/python3.5/site-packages/setuptools-19.4-py3.5.egg/pkg_resources/__init__.py", line 1072, in best_match
  File "/nix/store/86hcp0sb3ml9wfdfgapzn5bdvy1darh9-python3.5m-setuptools-19.4/lib/python3.5/site-packages/setuptools-19.4-py3.5.egg/pkg_resources/__init__.py", line 1084, in obtain
  File "/nix/store/86hcp0sb3ml9wfdfgapzn5bdvy1darh9-python3.5m-setuptools-19.4/lib/python3.5/site-packages/setuptools-19.4-py3.5.egg/setuptools/dist.py", line 380, in fetch_build_egg
  File "/nix/store/86hcp0sb3ml9wfdfgapzn5bdvy1darh9-python3.5m-setuptools-19.4/lib/python3.5/site-packages/setuptools-19.4-py3.5.egg/setuptools/command/easy_install.py", line 634, in easy_install
distutils.errors.DistutilsError: Could not find suitable distribution for Requirement.parse('setuptools-scm')
builder for ‘/nix/store/2x5rfglpbj27fa63fg1abv3jij4zbsbj-python3.5-execnet-1.4.1.drv’ failed with exit code 1
building path(s) ‘/nix/store/rv8s8xr5fakr18ngzrb3i7f8j5a8wvh5-python3.5-mccabe-0.4.0’
cannot build derivation ‘/nix/store/pkjrkrl874z6dfwj57j545igkc94k04h-python3.5-pytest-cache-1.0.drv’: 1 dependencies couldn't be built
killing process 67342
cannot build derivation ‘/nix/store/v48hcg7s73qamrlsksmf6j3lv8qarxbh-python3.5-vulnix-0.1.dev0.drv’: 1 dependencies couldn't be built
error: build of ‘/nix/store/v48hcg7s73qamrlsksmf6j3lv8qarxbh-python3.5-vulnix-0.1.dev0.drv’ failed
```

**Comment by plumps** [comment]
@garbas Do you have any idea how to tackle this issue?

**Comment by garbas** [comment]
add ``setuptools-scm`` as buildInputs for execnet

**Comment by plumps** [comment]
Didn't make cut, there is something wrong with the SSL certification system on mac

**Comment by garbas** [comment]
/me just got osx machine to give vulnix a try there

**Comment by plumps** [comment]
:+1:

**Comment by garbas** [comment]
fixed in #15
